### ISSUE TITLE
PHP notice when open eventlist without menu item

### ISSUE THREAD

**Issue by simokarpin** [jemproject/JEM-Project#901]
Code uses undefined constant - I guess that should be translation text?
Notice on page:
```php
Notice: Use of undefined constant COM_JEM_EVENTS - assumed 'COM_JEM_EVENTS' in /var/www/components/com_jem/views/eventslist/view.html.php on line 82

**Comment by JKoelman** [comment]
this one is happening as the language-string is not within a JText.

```$pagetitle	= $params->def('page_title', $menuitem ? $menuitem->title : COM_JEM_EVENTS);```

should be somethign like  ```$pagetitle	= $params->def('page_title', $menuitem ? $menuitem->title : JText::_('COM_JEM_EVENTS'));```

**Comment by Hoffi1** [comment]
Yep, that's wrong.
Think this should be fixed in JEM 2.1.3.
